### PR TITLE
Se 548 create documentation for access level0 after rename is finalized

### PR DESCRIPTION
**Pull request by TJ-Everett**

### CODE REVIEW

File: docs/userGuides/rules/ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Account Deny For No Access Level Rule
+
+## Purpose
+
+The purpose of this rule is to provide a way to prevent the transfer of assets without application verification. For example, the application may decide users may not accumulate or transfer any assets without first performing specific onboarding activities. The application developer may set the account deny for no access level rule to active. As accounts are introduced to the ecosystem, they may not receive any assets until the application changes their access level to a higher value.
```
Comment (ShaneDuncan602) [inline docs/userGuides/rules/ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md]: I think this first sentence should probably just refer to the Access Level itself: "without application verification" to "for accounts that do not have an access level or whose access level has been set to 0"

File: docs/userGuides/rules/ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+2. The processor checks that the access level is greater than 0. If the access level is equal to zero, then the transaction reverts.
+3. The application manager sends to the procotol's rule processor the access level of the `from` account in the transaction.
+4. The processor checks that the access level is greater than 0. If the access level is equal to zero, then the transaction reverts.
+
+###### *see [ApplicationAccessLevelProcessorFacet](../../../src/protocol/economic/ruleProcessor/ApplicationAccessLevelProcessorFacet.sol) -> checkAccountDenyForNoAccessLevel*
+
+## Evaluation Exceptions 
+- This rule doesn't apply when a **ruleBypassAccount** address is in either the *from* or the *to* side of the transaction. This doesn't necessarily mean that if an rule bypass account is the one executing the transaction it will bypass the rule, unless the aforementioned condition is true.
+- This rule doesn't apply when a **registeredAMM** address is in either the *from* or the *to* side of the transaction.
+- This rule doesn't apply when the *to* address is the zero address to allow for burning while rule is active.
```
Comment (ShaneDuncan602) [inline docs/userGuides/rules/ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md]: This is getting changed. We will be removing that burn check from the handler. The rule will not be applicable to a burn. It's in [ticket](https://thracklelabs.atlassian.net/browse/SE-565). Not sure how we should handle this. We could change it here or add something to the ticket to change it. Personally, I think we should go ahead and change it so it doesn't get missed.

File: docs/userGuides/rules/README.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
...
+| [Account Deny For No Access Level](./ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md) | The purpose of this rule is to provide a way to prevent the transfer of assets without application verification. For example, the application may decide users may not accumulate or transfer any assets without first performing specific onboarding activities. The application developer may set the account deny for no access level rule to active. As accounts are introduced to the ecosystem, they may not receive any assets until the application changes their access level to a higher value. |
```
Comment (ShaneDuncan602) [inline docs/userGuides/rules/README.md]: See above

**Review by ShaneDuncan602 — CHANGES_REQUESTED** [review]
Couple of itty bitty things

File: docs/userGuides/rules/ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md (hunk 2 above)
Comment (TJ-Everett) [reply docs/userGuides/rules/ACCOUNT-DENY-FOR-NO-ACCESS-LEVEL.md]: Agreed changing here and leaving a note to ensure the docs match after that ticket is complete.

**Review by ShaneDuncan602 — APPROVED** [review]
Perfect!

**Pull request merged**